components/images/CookiebotIcon.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <svg
+    version="1.1"
+    id="Layer_1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    x="0px"
+    y="0px"
+    viewBox="0 0 570.3 168.8"
+    style="enable-background: new 0 0 570.3 168.8"
+    xml:space="preserve"
+  >
+    <g>
+      <path
+        class="fill-gray-800 dark:fill-white"
```

Comment:
You can move these classes to the `<g>` tag. As they are all the same.

```suggestion
    <g class="fill-gray-800 dark:fill-white">
      <path
```